### ISSUE TITLE
"Cannot read property 'type' of null" in valid-aria-role.js

### ISSUE THREAD

**Issue by lencioni** [evcohen/eslint-plugin-jsx-a11y#4]
v0.5.0

with the following JSX

```jsx
function Foo() {
  return <Bar baz />;
}
```

I get the following error:

```
Cannot read property 'type' of null
TypeError: Cannot read property 'type' of null
    at EventEmitter.JSXAttribute (/path/to/project/node_modules/eslint-plugin-jsx-a11y/lib/rules/valid-aria-role.js:27:43)
```

However it works if I use this instead:

```jsx
function Foo() {
  return <Bar baz={false} />;
}
```

**Comment by lencioni** [comment]
Thanks for the speedy fix and release!

**Comment by evcohen** [comment]
no problem! thanks for reporting. waiting for travis to pass before i publish, may be a few mins.

**Comment by evcohen** [comment]
v0.5.1 published 😄
